src/machine/Coffee.java

```diff
@@ -0,0 +1,41 @@
+package machine;
+
+enum Coffee {
+    ESPRESSO(250, 0, 16, 1, 4),
+    LATTE(350, 75, 20, 1, 7),
+    CAPPUCCINO(200, 100, 12, 1, 6);
+
+    private final int amountOfWater;
+    private final int amountOfMilk;
+    private final int amountOfCoffeeBeans;
+    private final int amountOfDisposableCups;
+    private final int price;
+
+    Coffee(int amountOfWater, int amountOfMilk, int amountOfCoffeeBeans, int amountOfDisposableCups, int price) {
+        this.amountOfWater = amountOfWater;
+        this.amountOfMilk = amountOfMilk;
+        this.amountOfCoffeeBeans = amountOfCoffeeBeans;
+        this.amountOfDisposableCups = amountOfDisposableCups;
+        this.price = price;
+    }
+
+    int getAmountOfWater() {
+        return this.amountOfWater;
```

Comment:
Usually `this` keyword is not used in getters. It is usually used when there is a method/constructor parameter or a local variable with the same name as an instance field